instances/bodySystem/musculoskeletalSystem.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/musculoskeletalSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": null,
+  "description": "Anatomical system that consists of the muscular and skeletal systems.",
```

Comment:
definition: "The 'musculoskeletal system' is an anatomical organ system composed of organs providing the body with movement, stability, shape and support."

description: "The musculoskeletal system (sometimes also called locomotor system) is subdivided into two broader systems, the skeletal system and the muscular system. The skeletal system includes bones and joints. The muscular system includes all muscles in the body."